SignalRServiceBenchmarkPlugin/plugins/Plugin.Microsoft.Azure.SignalR.Benchmark/SignalRPlugin/MasterMethods/CreateConnection.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Common;
+using Plugin.Base;
+using Rpc.Service;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Plugin.Microsoft.Azure.SignalR.Benchmark.MasterMethod
+{
+    public class CreateConnection : IMasterMethod
+    {
+        public Task Do(IDictionary<string, object> stepParameters, IDictionary<string, object> pluginParameters, IList<IRpcClient> clients)
+        {
+            var success = true;
+
+            success = stepParameters.TryGetTypedValue(SignalRConstants.ConnectionTotal, out int connectionTotal, Convert.ToInt32);
```

Comment:
Is it possible to move PluginUtils.HandleGetValueResult into TryGetTypedValue? That method seems to be part of HandleGetValueResult.

Reply:
Make sense, thanks